aodh/meta/salt.yml

```diff
@@ -0,0 +1,6 @@
+orchestrate:
+  server:
+    priority: 690
+    batch: 1
+    require:
+    - salt: keystone.server
```

Comment:
Do we want that? I am not sure the `orchestrate` meta is still a thing… @cznewt?